assets/font/CertificateEncoderDecoder.svg

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   version="1.1"
+   viewBox="-10 0 689.33333 682.66666"
+   width="1034"
+   height="1024"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:dc="http://purl.org/dc/elements/1.1/">
+  <path
+     class="clr-i-outline clr-i-outline-path-1"
```

Comment:
Quick question I forgot to ask: where is this icon coming from? I should make sure its license allows us to use it.

Reply:
Good call - I sourced it from here: https://www.svgrepo.com/svg/372267/certificate and confirmed that it is provided via [MIT License](https://www.svgrepo.com/page/licensing#MIT).  The only stipulation I see there is:
> copyright notice – a copy of the license or copyright notice must be included with software

If this is a problem or too much hassle, please let me know and I can take a crack at a completely original one or find a different one.  I had to bring this one into Inkscape and do some tweaking, along with some of the steps in the [readme](https://github.com/veler/DevToys/blob/main/assets/font/README.md), so now I am well acquainted with this process.

Reply:
MIT License is fine. 😁 Please make sure to add their license to the Third Party Notice file, in the root of the repository.